### PR TITLE
TelegramNotifier

### PR DESCRIPTION
**Pull request by MiguelCarranza**
Simple [Telegram Bot](https://core.telegram.org/bots/api) Notifier

### CODE REVIEW

File: config-example.yaml
Comment (martomi) [inline config-example.yaml:19]: Please add `enable: false` config key.

**Review by martomi — CHANGES_REQUESTED** [review]
@MiguelCarranza looks good, thanks for the PR! Unfortunately I forgot to mention in CONTRIBUTION.md that PRs should be opened against the `dev` branch. I changed the target now - could you please rebase?

File: src/notifier/telegram_notifier.py
Comment (greimela) [inline src/notifier/telegram_notifier.py:29]: We could disable the notification in case of a low event priority.
```suggestion
                        "parse_mode": "Markdown",
                        "disable_notification": event.priority == EventPriority.LOW,
```

File: src/notifier/telegram_notifier.py
Comment (greimela) [inline src/notifier/telegram_notifier.py:28]: I would add a 🚨 symbol in front of the text in case of a HIGH event priority, since Telegram does not support sending high priority messages itself.

File: tests/notifier/test_telegram_notifier.py
Comment (greimela) [inline tests/notifier/test_telegram_notifier.py:10]: I would add some tests for the HIGH and NORMAL priority cases, if they get handled differently.

**Review by greimela — COMMENTED** [review]
Oh, I also created a TelegramNotifier yesterday evening, it looks quite similar 😄 

I will add some suggestions here, for things I added in my version.

File: src/notifier/telegram_notifier.py
Comment (martomi) [reply src/notifier/telegram_notifier.py:28]: Good idea!

File: config-example.yaml
Comment (MiguelCarranza) [reply config-example.yaml:19]: dang, I probably removed it when renaming the keys

**Comment by MiguelCarranza** [comment]
> @MiguelCarranza looks good, thanks for the PR! Unfortunately I forgot to mention in CONTRIBUTION.md that PRs should be opened against the `dev` branch. I changed the target now - could you please rebase?

Comments addressed and rebased!

**Comment by martomi** [comment]
Perfect thanks! Only nitpick I have left is to squash the commits to keep history cleaner, but I probably can also do it after rebasing a bit later today :-)

**Comment by greimela** [comment]
@martomi You could just use squash as merge strategy, I like to use that in my projects as well.

**Comment by martomi** [comment]
@greimela Thanks, I'll try that but I don't like that there's little control over the resulting commit message 😄

**Comment by greimela** [comment]
AFAIK you can freely edit the resulting commit message as maintainer. Per default it uses the pull request title, but you can modify it before merging.

**Comment by martomi** [comment]
Wow, that's very cool, thanks for the tip!

**Comment by martomi** [comment]
@greimela did you also use Telegram Bot for your implementation? I'm updating the README now and trying to understand how do you ensure that the bot cannot be subscribed by a third person?

**Pull request merged**
